src/sass/core/_print.scss

```diff
@@ -458,10 +458,15 @@ h4 {
 }
 
 .or-text-print-initialized {
-    input {
-        &[type=text] {
+    input[type=text], 
+    textarea {
+        &:not(.default-print) {
             display: none;
         }
+        
+        &.default-print {
+            overflow: hidden;
+        }
```

Comment:
I think we can clean up the amount of CSS classes used with this new plan.

I think .or-text-print-initialized no longer serves any purpose (now that we use printify and deprinty events to make temporary modifications during printing only).

I think the name `.default-print` is a little confusing. I'm wondering if we can use just `.print-replacement` (and no longer need `.print-only` and `.print-default`). Then we can target with `.print-only ~ input[type=text], .print-only ~ textarea{}`. Does that sensible to you?

And we could just remove the whole added element and add it only during the printify/deprintify events. Or toggle a .hide/.print-replacement class. Whatever you think is best.

Reply:
style updated